src/dryrun.ts

```diff
@@ -148,79 +149,7 @@ export async function createDryrun({
   });
 }
 
-interface StackValueResponse {
```

Comment:
This file's changes feel somewhat unrelated to the stated goal of the PR, can you walk me through the intentions here?

Reply:
Good question. This code broke because algod responses now contain bigints for certain fields. I could have just changed those fields, but I noticed a lot of the definitions here were redundant and no longer necessary, since we have fully typed algod responses.